include/envoy/http/codec.h

```diff
@@ -13,6 +13,9 @@
 namespace Envoy {
 namespace Http {
 
+// Based on the default limit in nghttp2.
```

Comment:
Originally we went with 60 to be well less than nghttp2's 63, but if we're documenting all over the nghttp2 is 63 saying we picked 60 because nghttp2 might be confusing.

Maybe something like noting the default is a legacy value designed to be safely under nghttp2's default limit?

Reply:
doneeee